### PR TITLE
fix(route/bjnews): Update to new standard

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bjnews/cat/depth?limit=2
/bjnews/cat/diyikandian?limit=2
/bjnews/cat/news?limit=2
/bjnews/column/204?limit=2
/bjnews/column/9179?limit=2
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Anti-bot or rate limit / 反爬/频率限制
  - [x] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/bjnews/cat.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/:cat',
+    categories: ['traditional-media'],
+    example: '/bjnews/depth',
+    parameters: {},
```
Comment (TonyRL) [inline lib/routes/bjnews/cat.ts:12]: Missing parameter description

File: lib/routes/bjnews/cat.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    features: {},
+    radar: [
+        {
+            source: ['www.bjnews.com.cn/:cat'],
+        },
+    ],
+    name: '分类',
+    maintainers: [''],
```
Comment (TonyRL) [inline lib/routes/bjnews/cat.ts:20]: Missing maintainer

File: lib/routes/bjnews/cat.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    const url = `https://www.bjnews.com.cn/${ctx.req.param('cat')}`;
+    const res = await ofetch(url);
+    const $ = load(res);
+    const list = $('#waterfall-container .pin_demo > a')
+        .toArray()
+        .map((a) => ({
+            title: $(a).text(),
+            link: $(a).attr('href'),
+        }))
+        .slice(0, ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')) : 15);
```
Comment (TonyRL) [inline lib/routes/bjnews/cat.ts:35]: slice them before mapping

File: lib/routes/bjnews/cat.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                const $d = load(responses);
+                // $d('img').each((i, e) => $(e).attr('referrerpolicy', 'no-referrer'));
+
+                const single = {
+                    ...item,
+                    pubDate: timezone(parseDate($d('.left-info .timer').text()), +8),
+                    author: $d('.left-info .reporter').text(),
+                    description: $d('#contentStr').html(),
+                };
+                return single;
```
Comment (TonyRL) [inline lib/routes/bjnews/cat.ts:50]: reuse `item` instead of spreading `item`

File: lib/routes/bjnews/cat.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    const url = `https://www.bjnews.com.cn/${ctx.req.param('cat')}`;
+    const res = await ofetch(url);
+    const $ = load(res);
+    const list = $('#waterfall-container .pin_demo > a')
+        .toArray()
+        .slice(0, ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')) : 15)
+        .map((a) => ({
+            title: $(a).text(),
+            link: $(a).attr('href'),
+            category: [`source:${$(a).parent().find('.source').text().trim()}`],
```
Comment (TonyRL) [inline lib/routes/bjnews/cat.ts:35]:
```suggestion
            category: $(a).parent().find('.source').text().trim(),
```

Comment (dzx-dzx) [reply lib/routes/bjnews/cat.ts:35]: I want to distinguish it with each page's own keywords, if there would be.

Comment (dzx-dzx) [reply lib/routes/bjnews/cat.ts:34]: ...No?

File: lib/routes/bjnews/column.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        guid: e.uuid,
+        pubDate: timezone(parseDate(e.row.publish_time), +8),
+        updated: timezone(parseDate(e.row.update_time), +8),
+        link: `https://www.bjnews.com.cn/detail/${e.uuid}.html`,
+    }));
+
+    const out = await Promise.all(list.map((item) => fetchArticle(item)));
+    return {
+        title: `新京报 - 栏目 - ${res.data[0].row.column_info[0].column_name}`,
+        link: url,
```
Comment (TonyRL) [inline lib/routes/bjnews/column.ts:39]: The link here should link to a human-readable webpage instead of API response

**Pull request merged**